Sections/EEA/GPPExtension: IAB Europe TCF.md

```diff
@@ -129,30 +135,48 @@ The core segment must always be present. It consists of the following fields:
       Note: With TCF v2.2 support for legitimate interest for purpose 3 to 6 has been deprecated. Bits 2 to 5 are required to be set to <code>0</code>.</td>
     </tr>
     <tr>
+  <tr style="background-color:#000;color:#FFF;">
+    <td colspan="4">
+        <strong>Specific Jurisdiction Disclosures</strong>
+    </td>
+  </tr>
   <td>PurposeOneTreatment</td>
   <td>Boolean</td>
-  <td>1 Purpose 1 was NOT disclosed at all.<br><br>0 Purpose 1 was disclosed commonly as consent as expected by the Policies.<br><br>CMPs can use the PublisherCC field to indicate the legal jurisdiction the publisher is under to help vendors determine whether the vendor needs consent for Purpose 1.</td>
+  <td>1 Purpose 1 was NOT disclosed at all.<br>0 Purpose 1 was disclosed commonly as consent as expected by the Policies.<br>CMPs can use the PublisherCC field to indicate the legal jurisdiction the publisher is under to help vendors determine whether the vendor needs consent for Purpose 1.</td>
```

Comment:
can we rephrase the logic of this. having a 1 as "no" and 0 as "yes" seems incorrect and may lead to errors. better something like:
0 = No special treatment. Purpose 1 was disclosed
1 = Special treatment. Purpose 0 was not disclosed

Reply:
Update accordingly here and in the TCF specification.

Reply:
Fixed